### ISSUE TITLE
Improve Visualization Load Time

### ISSUE THREAD

**Issue by vrajur-nd** [NextDroid/potree#13]
The development visualization code in https://github.com/NextDroid/potree/commit/f6c546686275f6cbbf623914e733c1097b2c3a7d takes a long time to initialize at start up (just loading tracked objects, vehicle mesh, rtk data, radar data and pointcloud). Loading detections takes a very long time, and loading gaps crashes the visualization (apparently by using up all the browser application's allocated memory). There is a need to profile the entire loading process to identify performance bottlenecks, potential memory management issues, and other optimization schemes with the goals of:
 - [ ] decreasing load time
 - [ ] enabling loading of "large' datasets (e.g. detections and gaps)

**Comment by calistenson** [comment]
@vrajur-nd wanna turn this into an epic and identify various experiments that would be useful to try in smaller tickets together at some point? I think once we have a list of the things we might try then we could start making progress on this ticket. For now I'll icebox it.

**Comment by vrajur-nd** [comment]
Yeah, I think icebox is good for now and I will make an epic for this to start adding to. In particular we need to reprofile things against recent versions of chrome, but that'll be incorporated into the future tests.
